src/Controller/RegionController.php

```diff
@@ -50,4 +50,27 @@ public function putRegion(Request $request, EntityManagerInterface $entityManage
 
         return $this->json(['status' => '200', 'message' => 'Região atualizada com sucesso.'], 200, ['Content-Type'=>'application/json; charset=utf-8']);
     }
+
+    #[Route('/region', name: 'korv_region_delete', methods: 'DELETE')]
+    public function deleteRegion(Request $request, EntityManagerInterface $entityManager): Response
+    {
+        $resultJson = json_decode($request->getContent(), true);
+        if ( !is_array($resultJson) ) {
+            return $this->json(['status' => '422'], 422, ['Content-Type'=>'application/json; charset=utf-8']);
+        }
+        if (!array_key_exists("id", $resultJson)) {
```

Comment:
Merge the two condition since you're returning the same error

Reply:
Okay, I fix that!